src/components/phase/phase.scss

```diff
@@ -4,8 +4,10 @@
 
 @mixin phase-banner() {
   padding-top: 10px;
-  padding-left: 22px;
+  padding-left: 15px;
   border-bottom: 1px solid $border-colour;
```

Comment:
Not sure whether this will make code more complicated if we use var() like https://www.w3schools.com/css/css3_variables.asp https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables  Just thought this is now so cumbersome whenever a minor change is required

Reply:
I have used the vars in most of the places wherever applicable, but in this case, since these styles are already defined, and to reduce complexity i have just modified the px as it is.